contributors/devel/e2e-tests.md

```diff
@@ -98,7 +103,11 @@ go run hack/e2e.go -- -v --up
 go run hack/e2e.go -- -v --test
 
 # Run tests matching the regex "\[Feature:Performance\]"
-go run hack/e2e.go -- -v --test --test_args="--ginkgo.focus=\[Feature:Performance\]"
+# Specify "--provider=local" flag when running the tests locally
+go run hack/e2e.go -- -v --test --test_args="--ginkgo.focus=\[Feature:Performance\]" --provider=local
+
+# Alternatively, specify it with KUBERNETES_PROVIDER
+KUBERNETES_PROVIDER=local go run hack/e2e.go -- -v --build --up --test --down
```

Comment:
@janetkuo I will remove these two lines as `KUBERNETES_PROVIDER=local` is deprecated.